src/app/charts/line-graph.component.ts

```diff
@@ -96,20 +96,23 @@ export class LineGraphComponent {
 
     // Set axis scales
     private setAxisScales(): void {
-        var parseTime = D3.timeParse('%Y-%m-%d');
+        // Time scales only recognize annual and daily data
+        var parseTime = D3.timeParse('%Y'); //D3.timeParse('%Y-%m-%d');
         this.extractedData.forEach(d => d.date = parseTime(d.date));
         this.xRange = D3.extent(this.extractedData, d => d.date);
         this.xScale.domain(this.xRange);
-        this.yScale.domain([0, D3.max(this.yData)]);
+        // Adjust y scale, prettify graph
+        let yPad = Math.abs((D3.max(this.yData) - D3.min(this.yData)) * 2/3);
```

Comment:
Numeric constants generally should be declared with a `const` assignment to ease reading and potential reuse.